stable/gemini/values.yaml

```diff
@@ -0,0 +1,16 @@
+image:
+  # pullPolicy -- imagePullPolicy - Highly recommended to leave this as `Always`
```

Comment:
```suggestion
  # image.pullPolicy -- imagePullPolicy - Highly recommended to leave this as `Always`
```

Reply:
That's what I get for copying your work 😜

Reply:
Definitely never copy my work :-D

I try not to even copy my work